app.py

```diff
@@ -17,7 +17,72 @@
 
 """This is the main script of the template project."""
 
-from template.version import __version__
+from typing import List, Optional
+
+import click
+import logging
+from github import Github
+from github.GithubException import GithubException
+from thoth.common import OpenShift
+
+from version import __version__, __title__
+
+_LOGGER = logging.getLogger(__title__)
+logging.basicConfig(level=logging.INFO)
+
+
+@click.command()
+@click.option(
+    "--repositories", help="Repositories to be analysed (e.g thoth-station/performance)",
```

Comment:
The invocation is not clear to me based on help. What if I provide `--repositories thoth-station/performance` and `--organizations aicoe`? Based on the code, it schedules two workflows for one repo and one organization. Why plural in that case?

And if I do not provide any of these, are organizations/repositories treated as empty strings down below?

Reply:
I've changed it to only have a set of unique names of repositories

If no org/repo is specified, it'll be now understood as an empty list, therefore nothing will be inspected.